deploy/bundle/manifests/workspace.devfile.io_devworkspaces.yaml

```diff
@@ -0,0 +1,5212 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    service.beta.openshift.io/inject-cabundle: "true"
+  creationTimestamp: null
+  labels:
+    app.kubernetes.io/name: devworkspace-controller
+    app.kubernetes.io/part-of: devworkspace-operator
+  name: devworkspaces.workspace.devfile.io
+spec:
+  conversion:
```

Comment:
Probably it should be moved to CSV instead https://olm.operatorframework.io/docs/advanced-tasks/adding-admission-and-conversion-webhooks/

Reply:
I was confused because the sample repo has `.spec.conversion` in the CRD but it's removed automatically when bundles are generated. Since we're not pointing operator-sdk at the kustomize templates repo, I don't think we get that benefit.

Reply:
From reading the documentation, I'm not exactly sure how this will work for us:

> In addition to these fields, OLM requires that you define:
> - The DeploymentName that OLM must mount the CA Cert information into, this should match the name of a deployment defined in the CSV

But the webhooks server deployment is not a part of the CSV. 

Also: Trying to implement this, I'm running into an issue with kustomize -- we need the name/namespace to apply to everything except the CSV, and I can't figure out how to have name/namespace generation disabled for specific types. Any ideas here?

Reply:
> But the webhooks server deployment is not a part of the CSV.

Pfffff, yeah, that's out of the olm case